README.md

```diff
@@ -106,41 +116,44 @@ This will export only those Export Collections that contain _any_ of the current
 These commands can all be bound to hotkeys or added to the Quick menu by right-clicking on them from the _Embark_ menu.
 
 ### Import
+
 Accessed from the _Embark -> Import_ menu item.
 
 Imports objects (FBX, OBJ, PLY) using hard-coded standard settings to enforce consistency.
 
 ### Spiral Curve
+
 Accessed from the _Embark -> Spiral_ menu item, or the 3D viewport's _Add -> Curve -> Spiral_ menu item.
 
 Adds a parametric spiral curve object to the scene.
 
 ### Connect Contextual
+
+[![Contributor Covenant](https://img.shields.io/badge/contributor%20covenant-v1.4-ff69b4.svg)](../CODE_OF_CONDUCT.md)
+
```

Comment:
This seems like the wrong place for this...

Reply:
It's the standard now for all Embark open source repos: https://github.com/EmbarkStudios/opensource-template/issues/20

Reply:
@lpil Looking at the rendered it's slightly outside of the contributing section.

<img width="1038" alt="Screenshot 2020-10-27 at 15 40 16" src="https://user-images.githubusercontent.com/4464295/97317360-033f9d00-186b-11eb-87e2-3ea7d279f4b1.png">

Reply:
Oops! I've copy pasted this so many times now my eyes have glazed over. 😵 
It's fixed now, thank you for catching this